.github/workflows/ci.yml

```diff
@@ -40,10 +40,13 @@ jobs:
           - "3.10"
         isForkBranch:
           - ${{ !contains(github.event.pull_request.head.repo.full_name, 'starburstdata') }}
+        isMaster:
+          - ${{ contains(github.ref, 'master') }}
         exclude:
           - engine: "starburst_galaxy"
             python: "3.9"
             isForkBranch: true
+            isMaster: false
```

Comment:
Why `isMaster`? Sometimes rebase can cause some changes to only occur once merged with latest master.

Reply:
Currently, Galaxy tests are not running on master.

Reply:
It would still trigger on isForkBranch `true` and isMaster `true`.

I think just using one condition seems simpler. Note that I changed into `isStarburstBranch`.

```
(github.event_name == 'pull_request' && contains(github.event.pull_request.head.repo.full_name, 'starburstdata')) || github.event_name != 'pull_request'
```